### ISSUE TITLE
关于GRPOTrainer的实现与论文中不同

### ISSUE THREAD

**Issue by Doctor-James** [Deep-Agent/R1-V#174]
想请教一下这里的实现为什么跟deepseek论文中的不同，这里为什么不是ref_per_token_logps.detach，并且没有clip
![Image](https://github.com/user-attachments/assets/b1382d70-628e-453b-880a-4e44e852f50c)

![Image](https://github.com/user-attachments/assets/a69d5e36-c17b-47fc-b466-533f2d4243f7)

open-r1中的实现与论文是相同的

![Image](https://github.com/user-attachments/assets/58468482-8004-40e2-b9f5-86424aa806e8)

**Comment by TIMMY-CHAN** [comment]
因为这个库一开始抄的就是open-r1-multimodal的库，open-r1-multimodal又是基于open-r1库写的，open-r1库一开始犯了这个错误，现在的版本中已经纠正这个问题了

**Comment by suruoxi** [comment]
根据上游的trl的[讨论](https://github.com/huggingface/trl/pull/2565#issuecomment-2595837761)来看，这不是一个bug？

**Comment by 0001Henry** [comment]
per_token_logps.detach()应该没问题，在num_iterations==1时等价于原公式；clip确实缺了
